docs/tutorial-aks-demo.md

```diff
@@ -0,0 +1,388 @@
+# Secure Secrets in AKS with Key Vault
+
+## Introduction
+
+A Secret in Kubernetes is meant to save sensitive and secure data like passwords, certificates, and keys. These Secret objects aren't secret or secure! They're encoded using Base64 and saved into *etcd*. Also, by default, any pod have access to them. 
+
+To secure these secrets, one solution could be to encrypt data at rest in Kubernetes as explained in [Kubernetes documentation](https://kubernetes.io/docs/tasks/administer-cluster/encrypt-data). But this feature (KMS) is not yet supported in AKS. However some work is in progress. Check the [AKS Roadmap](https://github.com/Azure/AKS/projects/1#card-36434441) and the [KMS Plugin for Key Vault](https://github.com/Azure/kubernetes-kms).
+
+Another solution would be using Azure Key Vault. It can securely encrypt sensitive data like keys, secrets, files, and certificates. Even more than just encryption, they offer powerful features like key rotation, expiration date and access policies.
+
+Now, to access Key Vault, a password is needed. Thus it will be non-securely saved in a Secret object in *etcd*! We returned back to the first problem. Fortunately in Azure, AKS can connect to Key Vault or SQL Database… using an Identity. This connection could be done using the open-source [project Pod Identity](https://github.com/Azure/aad-pod-identity). It uses Azure AD to create an Identity and assign the roles and resources.
```

Comment:
```suggestion
Now, to access Key Vault, a password is needed. Thus it will be non-securely saved in a Secret object in *etcd*! We returned back to the first problem. Fortunately in Azure, AKS can connect to Key Vault or SQL Database using an Identity. This connection could be done using the open-source [AAD Pod Identity](https://github.com/Azure/aad-pod-identity) project. It uses Azure AD to create an Identity and assign the roles and resources.
```